### DISCUSSION TITLE
Loss discrepancies when using the trainer

### DISCUSSION THREAD

**Discussion by martinez-zacharya** [Lightning-AI/pytorch-lightning#15442] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I'm encountering a weird training issue where my loss increases dramatically after one sample (like from 8 to 9.1188e+11) when I return the loss at the end of my training_step() function. However when I don't return anything (comment-out return(loss)), the loss remains stable for all of the samples.

Has anyone seen anything like this? Perhaps I am misunderstanding how lightning performs the training.

**Comment by martinez-zacharya** [comment, marked as answer]
Nevermind, my learning rate was set to 1e5 instead of 1e-5.
